packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1144,6 +1144,54 @@ export const formatters: Formatters = {
     format: ({ user, desktop_addr, directory_name, file_path, length }) =>
       `User [${user}] failed to write [${length}] bytes to file [${file_path}] in shared directory [${directory_name}] on desktop [${desktop_addr}]`,
   },
+  [eventCodes.DEVICE_CREATE]: {
+    type: 'device',
+    desc: 'Device Register',
+    format: ({ user, status }) =>
+      status.success
+        ? `User [${user.user}] has registered a device`
+        : `User [${user.user}] has failed to register a device`,
+  },
+  [eventCodes.DEVICE_DELETE]: {
+    type: 'device',
+    desc: 'Device Delete',
+    format: ({ user, status }) =>
+      status.success
+        ? `User [${user.user}] has deleted a device`
+        : `User [${user.user}] has failed to delete a device`,
+  },
+  [eventCodes.DEVICE_AUTHENTICATE]: {
+    type: 'device',
+    desc: 'Device Authenticate',
+    format: ({ user, status }) =>
+      status.success
+        ? `User [${user.user}] has successfully authenticated their device`
+        : `User [${user.user}] has failed to authenticate their device`,
+  },
+  [eventCodes.DEVICE_ENROLL]: {
+    type: 'device',
+    desc: 'Device Enrollment',
+    format: ({ user, status }) =>
+      status.success
+        ? `User [${user.user}] has successfully enrolled their device`
+        : `User [${user.user}] has failed to enroll their device`,
+  },
+  [eventCodes.DEVICE_ENROLL_TOKEN_CREATE]: {
+    type: 'device',
+    desc: 'Device Enroll Token Create',
+    format: ({ user, status }) =>
+      status.success
+        ? `User [${user.user}] created a device enroll token`
+        : `User [${user.user}] has failed to create a device enroll token`,
+  },
+  [eventCodes.DEVICE_ENROLL_TOKEN_SPENT]: {
+    type: 'device',
+    desc: 'Device Enroll Token Spent',
+    format: ({ user, status }) =>
+      status.success
+        ? `User [${user.user}] has used a device enroll token`
+        : `User [${user.user}] has failed to use a device enroll token`,
```

Comment:
Suggestion:

```suggestion
        ? `User [${user.user}] has spent a device enroll token`
        : `User [${user.user}] has failed to spend a device enroll token`,
```

so it highlights the fact that the token is gone and can't be spent again.